components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -106,15 +223,196 @@ class WidgetConfigurationPanel extends React.Component {
+                                      }}
+                                      checked={this.state.options[i].defaultData}
+                                      className="options-list"/>
+                        </div>);
+                        break;
+                    default :
+                        break;
+                }
+            }
+        }
+
+        return preferences;
+    }
+
+    getWidgetConfPanelContent() {
+        return (<div>
+            <List style={listStyle}>
+                <Subheader style={stylePanelHeader} className="options-list-header">Widget Configuration</Subheader>
+                <ListItem
+                    primaryText="Publishers"
+                    initiallyOpen={true}
+                    primaryTogglesNestedList={true}
+                    nestedItems={this.getPublishers()}
+                    nestedListStyle={itemBackgroundStyle}
+                    className="options-list-header"
+                    style={styleListHeader}
+                />
+                <Divider style={dividerStyle} inset={false}/>
+                <ListItem
+                    primaryText="Options"
+                    initiallyOpen={true}
+                    primaryTogglesNestedList={true}
+                    nestedItems={this.getPreferences()}
+                    nestedListStyle={itemBackgroundStyle}
+                    className="options-list-header"
+                    style={styleListHeader}
+                />
+            </List>
+        </div>);
+    }
+
+    WidgetConfDirtyNotifier() {
+        const actions = [
+            <FlatButton
+                label="No"
+                primary={true}
+                onClick={() => {
+                    this.props.handleCloseWithNo()
+                }}
+            />,
+            <FlatButton
+                label="Yes"
+                primary={true}
+                onClick={() => {
+                    this.props.handleCloseWithYes()
+                }}
+            />,
+        ];
+
+        return (
+            <div>
+                <Dialog
+                    title="You have unsaved Changes in Widget Configuration Panel"
+                    actions={actions}
+                    modal={true}
+                    open={this.props.isWidgetConfigPanelDirtyNotificationOpen}
+                >
+                    Do you want to save the changes ?
+                </Dialog>
+            </div>
+        );
+    }
+
     render() {
         return (<Drawer open={this.props.open} openSecondary={true}
                         containerStyle={styles.widgetDrawer}
                         containerClassName="widget-configuration-panel">
-            <div className="widget-configuration-panel-header"><FormattedMessage id="widget.configuration" defaultMessage="Widget Configuration"/></div>
-            <div><FormattedMessage id="publisher.list.heading" defaultMessage="Publishers"/></div>
-            {this.getPublishers()}
+            {/*<div>*/}
```

Comment:
Please remove these commented JSX if it's not required

Reply:
I purposefully left these along with the import statement at the top of the file in case someone wanted to revert the conf panel back to original, If you really need I can remove

Reply:
@Irindu Please remove all commented code. IF someone want to check , they can see via git commits.

Reply:
done with e751770d1edbd1fab609032d9e9e236eb8829ecf

Reply:
done with e751770d1edbd1fab609032d9e9e236eb8829ecf